src/lang/po/de/Prusa-Firmware-Buddy_de.po

```diff
@@ -427,23 +427,23 @@ msgstr "Meldungen"
 
 #: src/guiapi/include/MItem_menus.hpp:98
 msgid "Move Axis"
-msgstr "Bewegen Achsen"
+msgstr "Achsen Bewegen"
 
 #: src/gui/menu_vars.h:62
 msgid "Move E"
-msgstr "Bewegen E"
+msgstr "E Bewegen"
 
 #: src/gui/menu_vars.h:62
 msgid "Move X"
-msgstr "Bewegen X"
+msgstr "X Bewegen"
```

Comment:
`Bewege X` is correct and same as in Prusa Firmware
https://github.com/prusa3d/Prusa-Firmware/blob/MK3/lang/lang_en_de.txt#L653-L655